documentation/modules/mirrormaker/proc-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// assembly-mirrormaker.adoc
+
+[id='proc-mirrormaker-replication-{context}']
+= Synchronizing data between Kafka clusters
+
+Use {MirrorMaker} to synchronize data between Kafka clusters through configuration.
+
+The previous version of {MirrorMaker} continues to be supported.
+If you wish to use the resources configured for the previous version,
+they must be updated to the format supported by {MirrorMaker}.
+
+The configuration must specify:
+
+* Each Kafka cluster
+* Connection information for each cluster, including TLS authentication
+* The replication flow and direction
+** Cluster to cluster
+** Topic to topic
+* Replication rules
+* Committed offset tracking intervals
+
+Use the properties of the `KafkaMirrorMaker2` resource to configure your Kafka {MirrorMaker} deployment.
+
+You can configure access control for source and target clusters using TLS or SASL authentication.
+This procedure shows a configuration that uses TLS encryption and authentication for the source and target cluster.
+
+.Prerequisites
+
+* xref:cluster-operator-str[{ProductName} and Kafka is deployed]
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Edit the `spec` properties for the `KafkaMirrorMaker` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion} <1>
+  replicas: 3 <2>
+  connectCluster: "my-cluster-target" <3>
+  clusters: <4>
+  - alias: "my-cluster-source" <5>
+    bootstrapServers: my-cluster-source-kafka-bootstrap:9092 <6>
+  - alias: "my-cluster-target" <7>
+    bootstrapServers: my-cluster-target-kafka-bootstrap:9092 <8>
+    config: <9>
+      config.storage.replication.factor: 1
+      offset.storage.replication.factor: 1
+      status.storage.replication.factor: 1
+  mirrors: <10>
+  - sourceCluster: "my-cluster-source" <11>
+    targetCluster: "my-cluster-target" <12>
+    sourceConnector: <13>
+      config:
+        replication.factor: 1 <14>
+        offset-syncs.topic.replication.factor: 1 <15>
+        sync.topic.acls.enabled: "false" <16>
+    heartbeatConnector: <17>
+      config:
+        heartbeats.topic.replication.factor: 1 <18>
+    checkpointConnector: <19>
+      config:
+        checkpoints.topic.replication.factor: 1 <20>
+    topicsPattern: ".*" <21>
+    groupsPattern: "group1|group2|group3" <22>
```

Comment:
@scholzj - The original example _".*"_ was throwing the doc build, so I used this as an example. Maybe we use something else?